spring-kafka/src/main/java/org/springframework/kafka/requestreply/ReplyingKafkaTemplate.java

```diff
@@ -74,7 +77,13 @@
 
 	private long replyTimeout = DEFAULT_REPLY_TIMEOUT;
 
-	private volatile boolean schedulerSet;
+	private boolean schedulerSet;
+
+	private boolean sharedReplyTopic;
+
+	private String replyTopic;
+
+	private byte[] replyPartition;
```

Comment:
Can't they both be as `final`?
Plus can't we convert `topic` to `byte[]` immediately in the ctor instead of on each request?